packages/testsuite/cypress.config.ts

```diff
@@ -30,50 +34,156 @@ export default defineConfig({
                 },
               ])
               .withWaitStrategy(Wait.forLogMessage(new RegExp(".*(WildFly Full.*|JBoss EAP.*)started in.*")))
-              .withStartupTimeout(333000)
-              .withCommand(["-c", configuration || "standalone-insecure.xml"] as string[])
+              .withStartupTimeout(333000);
+            if (useNetworkHostMode === true) {
+              console.log("host mode");
+              findAPortNotInUse(8080, 8180)
+                .then((freePort) => {
+                  portOffset = freePort - 8080;
+                  wildfly
+                    .withNetworkMode("host")
+                    .withCommand([
+                      "-c",
+                      configuration || "standalone-insecure.xml",
+                      `-Djboss.socket.binding.port-offset=${portOffset.toString()}`,
+                    ] as string[]);
+                })
+                .catch((error) => {
+                  console.log(error);
+                });
+            } else {
+              console.log(`network mode, network name: ${config.env.NETWORK_NAME as string}`);
```

Comment:
> network mode

-> default network mode ?

Reply:
console log updated to `default network mode, network name: <name>`